### ISSUE TITLE
error plugin installation

### ISSUE THREAD

**Issue by n3xt-io** [FriendsOfShopware/FroshPlatformShareBasket#39]
Hello,

I wanted to use the Shopware plugin "shopping cart share" again after a while. Unfortunately, I get the following error during installation. 

Internal Server Error
An exception occurred while executing a query: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'DB_WHTSHOPWARE.frosh_share_basket_line_item' doesn't exist

Please help.

thank you.

**Comment by n3xt-io** [comment]
This exact plugin was installed in the past. I can't say for sure whether I deleted the tables manually at the time. I don't think so. Now with the new installation I've tried everything. With or without existing tables makes no difference. The error message was different. The plugin cannot be installed in the normal way.

**Comment by schneider-felix** [comment]
I'm not quite sure whether I understand you correctly. The tables are actually created when you install the plugin? Is that right?

Can you confirm that there is a `frosh_share_basket_line_item` table after you try to install the plugin?

**Comment by n3xt-io** [comment]
The table was not created with the plugin installation. After I took the table from an alternative installation I got a different error message. However, I had to edit these tables again because scripts from the FroshPlatformShareBasket/src/Migration folder caused errors during installation. Only after the original table had been created did the migration scripts run through and the plugin could be installed normally.
